stetho/src/main/java/com/facebook/stetho/inspector/elements/ShadowDocument.java

```diff
@@ -373,15 +373,17 @@ private void removeGarbageSubTree(
       // changes have been applied and after our caller (removeGarbageSubTree) removed another
       // element that claims this element as its child, then that means this element should not be
       // removed. It has been reparented, and recursion stops here.
-      if (elementInfo.parentElement != null &&
+      if (elementInfo != null && elementInfo.parentElement != null &&
```

Comment:
What about  wrapping below code into `elementInfo?.let { }`? I see that It will reduce number of `elementInfo != null`  expressions from two to one